history.txt

```diff
@@ -5,6 +5,55 @@
 OMERO version history
 =====================
 
+5.4.2 (January 2018)
+--------------------
+
+This is a bug-fix release.
+
+Improvements include:
+
+-  made projecting images belonging to another user only possible for users
+   with the required permissions to save the new images
+-  added documentation on a complete workflow for publishing data from
+   OMERO.server
+-  added references to the new OMERO pyramid format documentation (within the
+   OME Data Model and File Formats documentation)
+-  updated swingx library version used by OMERO.insight to stop insight-ij
+   plugin crashing in Fiji
+-  fixed issue with polygon and polyline ROIs created in OMERO.iviewer and 
```

Comment:
not included in 5.4.2

Reply:
Ah, i missed it was merged into workflows - it's labelled as milestone 5.4.2

Reply:
My fault. Like I said, I'll cross check ASAP.

Reply:
it was initially but I removed the label a while back